pkg/cli/mirror/release.go

```diff
@@ -365,6 +365,7 @@ func (o *ReleaseOptions) getMapping(opts *release.MirrorOptions) (image.TypedIma
 
 	opts.IOStreams.Out = file
 	opts.ToMirror = true
+	opts.KeepManifestList = true
```

Comment:
I'm wondering if there would be any impact for setting this to true for all architectures other than multi.
I don't think there is an impact... Just thought to mention it in case anyone thinks of anything.

Reply:
@sherine-k I tested mirroring `multi` release image in parallel with the single arch `amd64` and `arm64` release images, and didn't observe errors or any difference in the mirroring behavior.

Maybe @ardaguclu or @soltysh can confirm?